### PR TITLE
Fix: Qualification field for nurse user type

### LINKED ISSUE #8302: Qualification field for nurse user type
**Is your feature request related to a problem? Please describe.**
Nurses may be of various types based on qualification. Capturing and showing this in CARE is helpful. 

**Proposed Solution**

- [ ] User creation form (https://care.coronasafe.in/users/add) when user type is selected as "Nurse" add a field for "Qualification" with short text field. (the field is available for doctor user type already)

- [ ] In edit profile form (https://care.coronasafe.in/user/profile) , add the field so that the nurse can update/edit the qualification as needed.

- [ ] Show Qualification in user list (https://care.coronasafe.in/users?page=1&limit=18&user_type=Nurse) for nurses (follow the design from doctors listing where qualification is already there)

### PR DESCRIPTION
**Pull request by AdityaJ2305**
### Required Backends:

- https://github.com/ohcnetwork/care/pull/2514

## Proposed Changes

- Fixes #8302 
- Updated doctor_qualification field to qualification, making it applicable for the nurse user type as well.

## Screenshots
Screenshot 2024-10-03 at 12 28 58 PM
Screenshot 2024-10-03 at 12 29 54 PM
Screenshot 2024-10-03 at 12 52 48 PM

@ohcnetwork/care-fe-code-reviewers
@rithviknishad 

## Merge Checklist

- [x] Add specs that demonstrate bug / test a new feature.
- [x] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [x] Request for Peer Reviews

### CODE REVIEW

**Comment by nihal467** [comment]
Testing Note [ In Progress ]

- [x] user skill addition in user tab card
- [x] edit profile user skills

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — CHANGES_REQUESTED** [review]
@AdityaJ2305 The changes look good, please add i18n

**Comment by AdityaJ2305** [comment]
@khavinshankar  Sure

**Comment by nihal467** [comment]
@AdityaJ2305 can you push the request changes ASAP, since the related backend is merged, the frontend functionality is failing in develop

**Comment by AdityaJ2305** [comment]
Yes , Should i add  i18n for unknown text only and for all languages supported by care  ?

**Comment by rithviknishad** [comment]
https://github.com/ohcnetwork/care_fe#translations

**Comment by AdityaJ2305** [comment]
Done

File: src/Components/Users/ManageUsers.tsx [hunk 1]
```diff
@@ -282,16 +281,15 @@ export default function ManageUsers() {
                   {user.user_type === "Doctor" && (
                     <>
                       <div className="col-span-1">
-                        <UserDetails
-                          id="doctor-qualification"
-                          title="Qualification"
-                        >
-                          {user.doctor_qualification ? (
+                        <UserDetails id="qualification" title="Qualification">
```
Comment (khavinshankar) [inline src/Components/Users/ManageUsers.tsx]: i18n for the title

File: src/Components/Users/ManageUsers.tsx [hunk 2]
```diff
@@ -282,16 +281,15 @@ export default function ManageUsers() {
...
+                          {user.qualification ? (
                             <span className="font-semibold">
-                              {user.doctor_qualification}
+                              {user.qualification}
```
Comment (khavinshankar) [inline src/Components/Users/ManageUsers.tsx:290]: i18n

File: src/Components/Users/ManageUsers.tsx [hunk 3]
```diff
@@ -335,11 +337,27 @@ export default function ManageUsers() {
...
                   className={`${
                     isExtremeSmallScreen ? "flex flex-wrap" : "grid grid-cols-2"
                   }`}
                 >
+                  {user.user_type === "Nurse" && (
+                    <div className="row-span-1">
+                      <UserDetails id="qualification" title="Qualification">
+                        {user.qualification ? (
+                          <span className="font-semibold">
+                            {user.qualification}
```
Comment (khavinshankar) [inline src/Components/Users/ManageUsers.tsx:357]: 118n

File: src/Components/Users/ManageUsers.tsx [hunk 4]
```diff
@@ -335,11 +337,27 @@ export default function ManageUsers() {
...
                 )}
+
                 <div
                   className={`${
                     isExtremeSmallScreen ? "flex flex-wrap" : "grid grid-cols-2"
                   }`}
                 >
+                  {user.user_type === "Nurse" && (
+                    <div className="row-span-1">
+                      <UserDetails id="qualification" title="Qualification">
```
Comment (khavinshankar) [inline src/Components/Users/ManageUsers.tsx]: i18n for the title

File: src/Components/Users/UserAdd.tsx [hunk 5]
```diff
@@ -650,15 +658,17 @@ export const UserAdd = (props: UserProps) => {
               optionValue={(o) => o.id}
             />
 
+            {(state.form.user_type === "Doctor" ||
+              state.form.user_type === "Nurse") && (
+              <TextFormField
+                {...field("qualification")}
+                required
+                label="Qualification"
+                placeholder="Qualification"
```
Comment (khavinshankar) [inline src/Components/Users/UserAdd.tsx]: i18n

File: src/Components/Users/UserProfile.tsx [hunk 6]
```diff
@@ -727,15 +738,18 @@ export default function UserProfile() {
...
                           type="email"
                         />
+                        {(states.form.user_type === "Doctor" ||
+                          states.form.user_type === "Nurse") && (
+                          <TextFormField
+                            {...fieldProps("qualification")}
+                            required
+                            className="col-span-6 sm:col-span-3"
+                            label="Qualification"
+                            placeholder="Qualification"
```
Comment (khavinshankar) [inline src/Components/Users/UserProfile.tsx]: i18n

**Review by khavinshankar — CHANGES_REQUESTED** [review]
@AdityaJ2305 you missed i18n in few places

File: src/Components/Users/ManageUsers.tsx (hunk 2 above)
Comment (AdityaJ2305) [reply src/Components/Users/ManageUsers.tsx:290]: how can I do for this ,I can understand for static data like title placeholder label ?

Comment (rithviknishad) [reply src/Components/Users/ManageUsers.tsx:290]: ignore

**Review by rithviknishad — APPROVED** [review]

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**